tools/lsp/server/main.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"os"
+
+	"github.com/go-logr/logr"
+	"github.com/go-logr/zapr"
+	"github.com/tliron/glsp/server"
+	"go.uber.org/zap"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+var (
+	version = "0.0.1"
```

Comment:
this looks like something that we could derive from the go build info instead of using a const here. this even allows you to get git info and so on directly encoded if you dont build from a set tag